src/main/java/com/ita/if103java/ims/dto/WarehouseWidgetDto.java

```diff
@@ -0,0 +1,41 @@
+package com.ita.if103java.ims.dto;
+
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+
+@Component
+public class WarehouseWidgetDto {
+    @NotNull
+    private Long warahouseCapacity;
```

Comment:
Missed field for warehouseId And I would rename fields as 
- id
- capacity
- load

in order to keep code clean